packages/documentation/src/sidebar.js

```diff
@@ -99,15 +99,8 @@ module.exports = {
     },
     {
       id: 'visual-design',
-      name: 'Visual design',
-      href: '/visual-design',
-      indexmdx: true,
-      items: [
-        {
-          name: 'Formation components',
-          query: 'componentList',
-        },
-      ],
+      name: 'Components',
+      href: 'https://design.va.gov/components',
```

Comment:
From here they can click the link to go to Storybook, and Storybook will eventually replace the `components` page anyway.